modules/installation-gcp-regions.adoc

```diff
@@ -10,6 +10,7 @@
 You can deploy an {product-title} cluster to the following Google Cloud Platform (GCP)
 regions:
 
+* `africa-south1` (Johannesburg, South Africa)
```

Comment:
@brendan-daly-red-hat Please refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

I trust you will fix this issue of missing module file metadata, so I am proceeding ahead with the merge without blocking this PR.